Dockerfile

```diff
@@ -4,7 +4,7 @@
 # NOTES:            Currently this is one big dockerfile and non-optimal.
 
 #may include suffix
-ARG OPENSTUDIO_VERSION=3.1.0
+ARG OPENSTUDIO_VERSION=develop
```

Comment:
should this be 3.2.0 instead of develop?

Reply:
Yeah, we should pin this to 3.2.0-rc1. thanks for catching!